src/components/search/PageSearchModal/index.js

```diff
@@ -84,6 +86,7 @@ module.exports = React.createClass({
       page: page || 1,
       pageSize: this.props.pageSize,
       pageType: this.props.pageType,
+      groupValue: this.props.groupValues
```

Comment:
This is going to keep coming up. Can we add an issue to fix this after this PR has gone through?
I know @ellsclytn has to get it out quickly, and this was here before the work this PR concerns.
@slwen ?